meta-resin-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher

```diff
@@ -160,6 +160,21 @@ else
     inform "No need to flash second stage bootloader to a specific device."
 fi
 
+inform "Waiting for device registration to complete."
+# Wait for the device to register before copying config.json
+STARTTIME=$(date +%s)
+ENDTIME=$(date +%s)
+TIMEOUT=30
```

Comment:
@shaunmulligan @alexandrosm this will delay the device flashing by 30s if the device is unable to register in a flasher image (eg no network), how do you want to balance the tradeoff between waiting for the registration attempt and the ux of a no-network flash? (previously it was a 10s delay)

@willnewton is it possible to keep this check at/near the end of flashing and just recopy the config.json if it has changed in the meantime (or only copy the config.json once at the very end) so that we give the maximum time to register before blocking, allowing us to have a shorter blocking period whilst having the same overall time to register?

Reply:
Its probably best just to drop this patch from the series as it's unclear if it provides any benefit.